### PR TITLE
Add an Interactive Example for PhysicalShape

### PR DESCRIPTION
**Pull request by mghalayini**
This PR adds an interactive example to the [PhysicalShape API](https://api.flutter.dev/flutter/widgets/PhysicalShape-class.html).

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by Hixie** [comment]
test-exempt: only affects comments

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -1049,6 +1049,47 @@ class PhysicalModel extends SingleChildRenderObjectWidget {
...
+///     body: Center(
+///       child: PhysicalShape(
+///         elevation: 5.0,
+///         child: const SizedBox(
+///           child: Center(
+///             child: Text(
+///               'Hello, World!',
+///               style: TextStyle(
+///                 color: Colors.white,
+///                 fontSize: 20.0
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/basic.dart:1073]: nit: add a trailing comma

```suggestion
///                 fontSize: 20.0,
```

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -1049,6 +1049,47 @@ class PhysicalModel extends SingleChildRenderObjectWidget {
...
+///               ),
+///             ),
+///           ),
+///           height: 200.0,
+///           width: 200.0,
+///         ),
+///         clipper: ShapeBorderClipper(
+///           shape: RoundedRectangleBorder(
+///             borderRadius: BorderRadius.circular(10.0)
```
Comment (goderbauer) [inline packages/flutter/lib/src/widgets/basic.dart:1082]: nit: add trailing comma

```suggestion
///             borderRadius: BorderRadius.circular(10.0),
```

**Review by goderbauer — APPROVED** [review]
LGTM after nits are fixed

**Comment by mghalayini** [comment]
> LGTM after nits are fixed

Done.

**Pull request merged**